tests/integration/test_make_predictions.py

```diff
@@ -96,14 +98,49 @@ def test_run_server(
 
     base_url = "http://localhost:9001"
     with run_sap_integration_image(env_vars, f"{base_url}/v1/ping/"):
-        # Make predictions
         dataset = model_packages[model_package_id].get("model_dataset")
+        data_chunks = pd.read_csv(dataset, chunksize=MAX_ROWS_PER_REQUEST)
+
+        for chunk in data_chunks:
+            # Convert the chunk to a CSV byte stream
+            csv_data = BytesIO()
```

Comment:
Nit: the name csv_data would seem to indicate the whole thing rather than chunks.